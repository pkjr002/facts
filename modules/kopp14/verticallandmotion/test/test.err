mv: cannot stat ‘*globalsl*’: No such file or directory
mv: cannot stat ‘*.nc’: No such file or directory
