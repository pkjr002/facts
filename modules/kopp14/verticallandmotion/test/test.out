Initiating test in /scratch/rk509/test.1668002812...

Extracting data files...

Executing workflow...
python kopp14_preprocess_verticallandmotion.py --pipeline_id verticallandmotion-kopp14-verticallandmotion
python kopp14_fit_verticallandmotion.py --pipeline_id verticallandmotion-kopp14-verticallandmotion
python kopp14_project_verticallandmotion.py --pipeline_id verticallandmotion-kopp14-verticallandmotion
python kopp14_postprocess_verticallandmotion.py --pipeline_id verticallandmotion-kopp14-verticallandmotion --nsamps 500 --seed 4321 --baseyear 2005 --pyear_start 2020 --pyear_end 2300 --pyear_step 10 --locationfile /home/rk509/facts/modules/kopp14/verticallandmotion/test/../../../../scripts/moduletest/location_list.lst

Collecting output files...
verticallandmotion-kopp14-verticallandmotion_localsl.nc

Cleaning up...
